doc/changelog.md

```diff
@@ -115,6 +115,24 @@
       m = measure(0, postselect=1)
       return qml.expval(qml.PauliZ(0))
   ```
+* Add support for reset option in mid-circuit measurements.
+  [(#507)](https://github.com/PennyLaneAI/catalyst/pull/507)
+
+  This is an example of reset usage:
+
+  ```py
+  import pennylane as qml
+  from catalyst import qjit
+
```

Comment:
I don't think we need to include these imports for every example in changelog. 
```suggestion
```

Reply:
Done